.github/workflows/ci.yml

```diff
@@ -58,12 +58,33 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        target: [x86_64-unknown-linux-gnu, armv7-unknown-linux-gnueabihf, x86_64-apple-darwin, x86_64-pc-windows-gnu]
         include:
-          - os: ubuntu-latest
+
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
             e2e: true
-          - os: macos-latest
+            test_features: testcontainers
+            cnd_binary_path: target/debug/cnd
```

Comment:
if you path `--target` when building, even if it's not cross-compiling then the binary will endup in the `target/<target>/(debug|release)/` path.

So you only need a `binary_suffix` or `binary-ext` option to pass `.exe` for windows.

Reply:
Oh cool, that is good to know! I was annoyed at having to do this :D